### ISSUE TITLE
DNS Shark should timeout when a DNS Name Server does not respond.

### ISSUE THREAD

**Issue by jmiiller** [jmiiller/dns_shark#1]
It would be nice if DNS Shark timed out 30 seconds after it has no received a response from a DNS Name Server. 

This would provide the user with information concerning why they are not receiving a response from DNS Shark.

**Comment by danielhoherd** [comment]
Agreed. I ran into this problem today, expecting there to be a default timeout value of 10s or something. It would be great if this was implemented as a flag with a default value of something in 10-60s.
